index.html

```diff
@@ -7,9 +7,96 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
     <link rel="stylesheet" href="./styles/styles.css">
+    <script src="https://kit.fontawesome.com/279611ab89.js"
+    crossorigin="anonymous"></script>
 </head>
 
 <body>
+    <header>
+        <a class="logo wrapper" href="https://www.youtube.com/c/rupaulsdragrace/featured">
```

Comment:
interesting to add the wrapper class on an anchor tag! I've never done it like this but for our purposes that definitely works!